src/hobbits-widgets/bitcontainertreemodel.cpp

```diff
@@ -18,7 +18,31 @@ QVariant BitContainerTreeModel::data(const QModelIndex &index, int role) const
 
     BitContainer *container = static_cast<BitContainer*>(index.internalPointer());
     if (role == Qt::DisplayRole) {
-        return QVariant(container->name());
+        qint64 size = container->size();
+        float f_size;
+        QString unit = "bits";
+        if (size < 1000) {
+
+            f_size = size;
+            //QString unit = "bits";
+        } else if (size >= 1000 && size < 8000) {
+
+            unit = "bytes";
```

Comment:
A few (optional) things here.

- I think "bytes" could just be displayed as `B`
- maybe bytes should be used for all containers less than 1000 bytes that are divisible by 8, and bits are used otherwise
- decimals should be clamped to 2-3 characters